src/styles/blocks/catalog.scss

```
@@ -0,0 +1,16 @@
+.container {
+  padding: 50px 0 70px;
+  display: grid;
+  grid-template-columns: repeat(4, auto);
+  justify-content: left;
+  gap: 48px;
+  @media (max-width: 1024px) {
+    grid-template-columns: repeat(3 , auto);
+  }
+  @media (max-width: 768px) {
```

```suggestion
  gap: 48px;

  @media (max-width: 1024px) {
    grid-template-columns: repeat(3 , auto);
  }

  @media (max-width: 768px) {
```
add empty lines between media-queries to increase readibility